vkubelet/vkubelet.go

```diff
@@ -136,11 +144,11 @@ func (s *Server) Stop() {
 
 // reconcile is the main reconciliation loop that compares differences between Kubernetes and
 // the active provider and reconciles the differences.
-func (s *Server) reconcile(ctx context.Context) {
+func (s *Server) reconcile(ctx context.Context, clearStaleProviderPodsOnly bool) {
```

Comment:
Would it make sense to break out stale pod reconciliation into a separate function?